README.md

```diff
@@ -1,11 +1,11 @@
 # React Search Route
 
-...
+This library provides the ability to render children based off the existence of what resides in a [querystring](https://developer.mozilla.org/en-US/docs/Web/API/URL/search). Out of the box, [react-router-dom](https://github.com/ReactTraining/react-router) does not support routing via query parameters for a [plethora of reasons](https://github.com/ReactTraining/react-router/issues/4410#issuecomment-276400992). So we decided to form an opinion and encapsulate querystring routing logic ourselves.
```

Comment:
>So we decided to form an opinion...

Seems a little sassy, maybe?

Reply:
Maybe haha. But the base library chooses to not form an opinion on this.